pkgs/misc/drivers/openzwave/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub
+, doxygen, libxml2, git, which, pkgconfig, graphviz-nox
+, udev }:
+
+let
+  version = "2018-04-04";
+
+in stdenv.mkDerivation rec {
+  name = "openzwave-${version}";
+
+  src = fetchFromGitHub {
+    owner = "OpenZWave";
+    repo = "open-zwave";
+    rev = "ab5fe966fee882bb9e8d78a91db892a60a1863d9";
+    sha256 = "0yby8ygzjn5zp5vhysxaadbzysqanwd2zakz379299qs454pr2h9";
+  };
+
+  nativeBuildInputs = [ doxygen libxml2 git which pkgconfig graphviz-nox ];
+  buildInputs = [ udev ];
+
+  preBuild = ''
+    makeFlagsArray+=("DESTDIR=$out")
+    makeFlagsArray+=("PREFIX=")
+    makeFlagsArray+=("pkgconfigdir=lib/pkgconfig")
+    makeFlagsArray+=("sysconfigdir=etc/openzwave")
+  '';
```

Comment:
@dotlambda At least I've got "rid" of the prefix and lots of file moving in the fixup, but still not sure how to use `makeFlags`... At least it's quite alot cleaner overall now....